platform/security/certificate/security-ocsp-checker/src/main/java/org/codice/ddf/security/ocsp/checker/OcspChecker.java

```diff
@@ -446,6 +474,191 @@ private void postErrorEvent(String errorMessage) {
+              "        issuerNameHash: "
+                  + getValueOrDefault(Arrays.toString(cert.getIssuerNameHash()), "")
+                  + "\n");
+          logbuff.append(
+              "        issuerKeyHash: "
+                  + getValueOrDefault(Arrays.toString(cert.getIssuerKeyHash()), "")
+                  + "\n");
+          logbuff.append(
+              "        cert serial number: "
+                  + getValueOrDefault(cert.getSerialNumber(), "").toString()
+                  + "\n");
+        }
+      }
+    }
+    LOGGER.trace(logbuff.toString());
+  }
+
+  private void logResponse(OCSPResp response) {
+    BasicOCSPResp basicOCSPResp;
+    BasicOCSPResponse basicOCSPResponse;
+    try {
+      basicOCSPResp = ((BasicOCSPResp) response.getResponseObject());
+      basicOCSPResponse =
+          BasicOCSPResponse.getInstance(
+              ((BasicOCSPResp) response.getResponseObject()).getEncoded());
+      StringBuffer logbuff = new StringBuffer();
+      logbuff.append("OCSP Response: \n");
+      logbuff.append("  responseStatus: " + getValueOrDefault(response.getStatus(), "") + "\n");
+      logbuff.append("  responseBytes:\n");
+      logbuff.append(
+          "  responseType: "
+              + getValueOrDefault(basicOCSPResponse, "").getClass().getSimpleName()
+              + "\n");
+      logbuff.append("    response:\n");
+      logbuff.append("      tbsResponseData:\n");
+      if (basicOCSPResponse.getTbsResponseData() != null) {
+        logbuff.append(
+            "        version: "
+                + getValueOrDefault(basicOCSPResponse.getTbsResponseData().getVersion(), "")
+                    .toString()
+                + "\n");
+        logbuff.append("        responderId:\n");
+        if (basicOCSPResponse.getTbsResponseData().getResponderID() != null) {
+          logbuff.append(
+              "          byName: "
+                  + getValueOrDefault(
+                          basicOCSPResponse.getTbsResponseData().getResponderID().getName(), "")
+                      .toString()
+                  + "\n");
+          logbuff.append(
+              "          byKey: "
+                  + getValueOrDefault(
+                      Arrays.toString(
+                          basicOCSPResponse.getTbsResponseData().getResponderID().getKeyHash()),
+                      "")
+                  + "\n");
+        } else {
+          logbuff.append("          byName:\n");
+        }
+        if (basicOCSPResponse.getTbsResponseData().getProducedAt() != null) {
+          logbuff.append(
+              "        producedAt: "
+                  + getValueOrDefault(
+                      basicOCSPResponse.getTbsResponseData().getProducedAt().getTimeString(), "")
+                  + "\n");
+        } else {
+          logbuff.append("        producedAt:\n");
+        }
+      }
+      logbuff.append("        responses:\n");
+      if (basicOCSPResp.getResponses() != null) {
+        int i = 0;
```

Comment:
✏️ Same comment here regarding an index-based `for` loop.

Reply:
Fixed.